File: math-algorithm/11.php
<?php

// 3.4.3
$n = trim(fgets(STDIN));
$a = explode(' ', trim(fgets(STDIN)));
$b = explode(' ', trim(fgets(STDIN)));

$answer = 0;
for ($i = 0; $i < $n; $i++) {
  $result = ((int)$a[$i] * (1 / 3)) + ((int)$b[$i] * (2 / 3));
  $answer += $result;
}

echo $answer;

File: math-algorithm/11.py
# 3.4.3
N = int(input())
A = list(map(int, input().split()))
B = list(map(int, input().split()))

Answer = 0
for i in range(N):
  Result = (A[i] * (1 / 3)) + (B[i] * (2 / 3))
  Answer += Result

print('%.12f' % Answer)
